addons/web/static/src/views/calendar/hooks.js

```diff
@@ -114,13 +114,15 @@ export function useFullCalendar(refName, params) {
         }
     });
 
-    let isWeekendVisible = params.isWeekendVisible;
+    let isWeekendVisible = params.weekends;
     onWillUpdateProps((np) => {
         isWeekendVisible = np.isWeekendVisible;
     });
     onPatched(() => {
         instance.refetchEvents();
         instance.setOption("weekends", isWeekendVisible);
+        instance.destroy();
+        instance.render();
```

Comment:
This can't be done in `onWillUpdateProps` if the props `isWeekendVisible` is changed to avoid useless re-rendering ?

Reply:
I have done it in `onPatched` hook.